test/system/030-run.bats

```diff
@@ -1399,4 +1399,19 @@ search               | $IMAGE           |
     run_podman rm -f -t0 $cid
 }
 
+@test "podman run - rm pod if container creation failed with -pod new:" {
+    run_podman run -d --name foobar $IMAGE hostname
+    cid=$output
+
+    podname=pod$(random_string)
+    run_podman 125 run --rm --pod "new:$podname" --name foobar $IMAGE hostname
+    is "$output" ".*creating container storage: the container name \"foobar\" is already in use by"
+
+    # pod should've been cleaned up
+    # if container creation failed
+    run_podman 1 pod exists $podname
+
+    run_podman rm -f $cid
```

Comment:
Please add `run_podman rmi $(pause_image)`. This prevents nasty red warning messages in CI log output. And (less important) please remove the unnecessary `-f` from `rm`.

Reply:
Does it make sense to add `run_podman rmi $(pause_image)` in teardown?

Reply:
Not in `030-run.bats`: that set of tests hardly uses pods at all. `200-pod.bats` runs it in teardown.

You're right, though, this is a question that could've gone either way. At design time I chose to force individual tests to leave clean state.  I still lean somewhat in that direction, so, thank you for adding the `rmi` here.